ReactWindows/ReactNative.Shared/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -1,6 +1,10 @@
 ﻿using System;
 using System.Threading.Tasks;
+#if WINDOWS_UWP
 using Windows.Storage;
+#else
+using PCLStorage;
+#endif
```

Comment:
If we have the dependency anyway, we should just use it here.

---
In reply to: [85593448](https://github.com/ReactWindows/react-native-windows/pull/835#discussion_r85593448) [](ancestors = 85593448)